src/chronos/boto_utils.py

```diff
@@ -110,6 +111,11 @@ def cache_model_from_s3(
 
     # Use CloudFront CDN for faster, cached downloads if available
     if cloudfront_url:
+        warnings.warn(
+            f"Loading {s3_uri} from CloudFront is deprecated and will be removed in a future version. "
+            f"Please specify a HuggingFace model_id instead.",
+            category=DeprecationWarning,
```

Comment:
Use `FutureWarning` (deprecation warnings are never shown) and set `stack_level`